base/applications/cmdutils/attrib/attrib.c

```diff
@@ -72,7 +72,7 @@ ErrorMessage(
                        NULL, dwErrorCode, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
                        (LPWSTR)&szError, 0, NULL))
     {
-        ConPrintf(StdErr, L"%s %s\n", szError, szMessage);
+        ConPrintf(StdOut, L"%s %s\n", szError, szMessage);
```

Comment:
Why?

Reply:
See `modules/rostests/apitests/cmd/cmd.c`, please.
The `attrib` program doesn't generate standard error messages.